src/notebook_index.ipynb

```diff
@@ -251,13 +252,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Contact info\nGeneral, background and overview information is available at the [IDAES main website](https://idaes.org).\nFramework development happens at our GitHub repo where you can report issues/bugs or make contributions.\nFor further enquiries, send an email to: idaes-support@idaes.org\n"
+    "## Contact info\n",
+    "General, background and overview information is available at the [IDAES main website](https://idaes.org).\n",
+    "Framework development happens at our GitHub repo where you can report issues/bugs or make contributions.\n",
+    "For further enquiries, send an email to: idaes-support@idaes.org\n"
    ]
   }
  ],
  "metadata": {
-  "kernel_info": {}
+  "kernel_info": {},
```

Comment:
Similarly, should these changes be here?

Reply:
Not sure why/how this got changed. It was not intentional...